### PR TITLE
Add service name info for docker outside paasta

### PR DESCRIPTION
**Pull request by dichiarafrancesco**
If a docker container is outside paasta the MESOS_TASK_ID env variable is not available.  So to have the service name in our metrics we can add a new env variable in the Docker file called SERVICE_NAME and set the name we prefer.

Inside the Dockerfile you need to add this line:
`ENV SERVICE_NAME my_service_name`

### CODE REVIEW

File: src/fullerite/collector/docker_stats.go [hunk 1]
```diff
@@ -157,6 +158,8 @@ func getServiceDimensions(container *docker.Container) map[string]string {
 			tmp["service_name"] = serviceName
 			tmp["instance_name"] = instance
 			break
+		} else if envArray[0] == serviceNameLabel {
+			tmp["service_name"] = envArray[1]
```
Comment (tsheasha) [inline src/fullerite/collector/docker_stats.go]: Does envArray[1] require sanitization?

**Comment by gnufied** [comment]
@dichiarafrancesco should we update the corresponding unit test that tests this behaviour as well? It looks like pretty simple to do that.

File: src/fullerite/collector/docker_stats.go (hunk 1 above)
Comment (dichiarafrancesco) [inline src/fullerite/collector/docker_stats.go]: no i don't think so, because it's a name that the developer choose

Comment (tsheasha) [inline src/fullerite/collector/docker_stats.go]: @dichiarafrancesco that is even a bigger reason for this to be sanitised .. we don't want any name a developer chooses to go into our dimension values

Comment (gnufied) [inline src/fullerite/collector/docker_stats.go]: +1 . We might have to apply similar sanitization rules we are applying in meteorite etc to get rid of problematic stuff.

Comment (tsheasha) [inline src/fullerite/collector/docker_stats.go]: unless we want to move sanitisation to the metrics class itself? in the ad dimension function?
@dichiarafrancesco  do you think it makes more sense to add it there?

Comment (dichiarafrancesco) [inline src/fullerite/collector/docker_stats.go]: @gnufied @tsheasha you're right.

**Comment by tsheasha** [comment]
tests are failing

**Comment by poros** [comment]
Isn't this too yelpy? Shouldn't we make it configurable?

**Comment by dichiarafrancesco** [comment]
@poros i didn't get it. can you be more detailed?

**Comment by tsheasha** [comment]
@dichiarafrancesco I think @poros means that the logic for the paasta specific docker containers should not be here, since this is a generic collector, it should be part of the config somehow.

**Comment by poros** [comment]
I was thinking to something like defining the list of env variables names you want to export as dimensions, but I didn't give much thought to how to actually implement this.

**Comment by tsheasha** [comment]
@poros actually coming to think of it, it is not really yelp-y since paasta is open source now and users of paasta might need to make this collector work for them .. for me it is a +1

**Comment by poros** [comment]
Well, not that we can't refine it later on, if the issue ever arise. It was just a small concern. Feel free to go ahead, if reviewers support the change.

**Comment by dichiarafrancesco** [comment]
@poros we can open another pull request for it, if it makes sense. right now i think the collector has everything you need. but since i like your idea let's discuss about it.

**Pull request merged**